lib/themes/dosomething/paraneue_dosomething/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@dosomething/forge": "~6.7.0",
-    "dosomething-modal": "~0.3.0",
+    "dosomething-modal": "0.3.3",
```

Comment:
the hell... https://github.com/DoSomething/phoenix/blob/dev/lib/themes/dosomething/paraneue_dosomething/package.json#L22